gandalf.js

```diff
@@ -1,125 +1,134 @@
-const uniqueSortedArray = [...new Set(array)].sort();
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?